lib/jekyll.rb

```diff
@@ -78,7 +78,7 @@ class << self
     # images and allows you to skip that when working in development.
 
     def env
-      ENV["JEKYLL_ENV"] || "development"
+      ENV["JEKYLL_ENV"] || "local"
```

Comment:
I think I have a different feel for the user base, I guess. I see many more front end developers who have no concept of deployment environments. They know what is building their site (their local computer, github, or some other service), but `development` and `production` would be foreign ideas to them.